.github/workflows/publish_to_anaconda.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish to Anaconda
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-to-anaconda:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache Conda
+        uses: actions/cache@v2
+        env:
+          # Increase this value to reset cache if conda/e3sm_diags_env_dev.yml has not changed in the workflow
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-publish
+
+      - name: Set up Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: "e3sm_diags_publishing"
+          channel-priority: strict
+          python-version: 3.7
+          auto-update-conda: true
+          # IMPORTANT: This needs to be set for caching to work properly!
+          use-only-tar-bz2: true
+
+      - name: Additional Conda Config
+        run: |
+          conda install anaconda-client conda-build conda-verify
+          conda config --set anaconda_upload no
+
+      - name: Build Conda Package
+        run: conda build -c conda-forge --output-folder . .
+
+      - name: Publish to Anaconda
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
+        run: anaconda upload --label main noarch/*.tar.bz2 --force
```

Comment:
hey Tom, I remembered that it needs '-u e3sm' to publish to e3sm channel. but it is not included here?

Reply:
Thanks Jill, `nightly_conda_upload.sh` had `-u uvcdat` but that's outdated. 

I'll include `-u e3sm`, which I'm guessing needs to match the GitHub account for permission to publish on the e3sm channel.

Reply:
Actually, I don't think `-u e3sm` is required since you have the `ANACONDA_API_TOKEN` env variable, which points the upload to the e3sm channel.

`anaconda-upload -h`
```
  -u USER, --user USER  User account or Organization, defaults to the current user
```

Reply:
Actually, I don't think `-u e3sm` is required since you have the `ANACONDA_API_TOKEN` env variable, which points the upload to the e3sm channel

Reply:
Awesome! I don't have other comments. I think you can go ahead and merge.